README.md

```diff
@@ -40,9 +40,26 @@ selenium-standalone start -- -role hub
 selenium-standalone start -- -role node -hub http://localhost:4444/grid/register
 selenium-standalone start -- -role node -hub http://localhost:4444/grid/register -port 5556
 
+# specify options in config file
+selenium-standalone install --config=/path/to/config.json
+selenium-standalone install --config=./config/seleniumConfig.js
```

Comment:
Can you change this line to `selenium-standalone start ...` just to make it more explicit that the config file is supported in both actions.